src/wasm/kotlin/src/arcs/Particle.kt

```diff
@@ -99,8 +99,8 @@ abstract class Particle : WasmObject() {
 }
 
 abstract class Handle : WasmObject() {
-    var name: String? = null
-    var particle: Particle? = null
+    lateinit var name: String
+    lateinit var particle: Particle
```

Comment:
I'd consider putting these in the constructor for the Handle class rather than making them lateinit, since that will require extending classes to know that they must initialize those values - but nothing is forcing it:

```kotlin
abstract class Handle(
  var name: String,
  var particle: Particle
) : WasmObject() {
  abstract fun sync(encoded: String)
  abstract fun update(added: String, removed: String)
}
```

an alternative could be to make `name` and `particle`, `abstract val`s

Reply:
That's a more substantial change. Let me try it at a later stage when I understand this part of the codebase better!

Reply:
Sounds good!